webapp/src/Components/ReplaysSavedGroup/GroupPlayerStatsTableWrapper.tsx

```diff
@@ -0,0 +1,53 @@
+import {Tab, Tabs} from "@material-ui/core"
+import * as React from "react"
+import {GroupPlayerStatsResponse} from "../../Models/Replay/Groups"
+import {GroupPlayerStatsTable} from "./GroupPlayerStatsTable"
+import {GroupStatsButtons} from "./Shared/GroupStatsButtons"
+
+interface State {
+    selectedTab: string
+}
+
+interface Props {
+    stats: GroupPlayerStatsResponse
+}
+
+export class GroupPlayerStatsTableWrapper extends React.Component<Props, State> {
+    constructor(props: Props) {
+        super(props)
+        this.state = {selectedTab: "(per Norm Game)"}
+    }
+
+    public render() {
+        const playerStats = this.props.stats.playerStats
+
+        return (
+            <>
+                <Tabs value={this.state.selectedTab} onChange={this.handleTabChange}>
+                    {Object.keys(playerStats[0].stats)
+                        .reverse()
+                        .map((tab) => (
+                            <Tab label={tab} value={tab} key={tab} />
+                        ))}
+                </Tabs>
+                <GroupStatsButtons style={{display: "none"}} />
+                <GroupPlayerStatsTable
+                    stats={{
+                        playerStats: playerStats.map((player) => {
+                            return {
```

Comment:
can remove unnecessary {} (i.e. () => ({}) )

Reply:
Ah the parentheses were missing. I tried `() => {}` where `{}` was the object and realized I was dumb and it thought it was a method.